apps/docs/src/components/document-renderer/components/base-tags/blockquote.tsx

```diff
@@ -175,7 +173,7 @@ export const Blockquote = (props: BlockquoteProps) => {
             ListItem
           );
 
-          if (listItemCount > 1) {
```

Comment:
Without this change, blockquotes with one bullet child would not render like so:
![image](https://github.com/user-attachments/assets/7a6feb8f-3c24-46a6-8227-00377de6893b)